### ISSUE TITLE
Move camera position without changing camera direction

### ISSUE THREAD

**Issue by stam** [yomotsu/camera-controls#317]
### Is your feature request related to a problem? Please describe.

Hey, first of all thanks for this library.

We are implementing our own dollyToCursor, where we want to dolly towards/away from a point in 3d space.

### Describe the solution you'd like

We want to move the camera along a line from the oldCameraPosition to that dollyTarget.

The problem is: we can't simply change the camera position, as it will then change the cameraDirection to look at the old target.

### Describe alternatives you've considered

As a workaround we tried to set the cameraTarget to the oldTarget + cameraTranslation.
This works, until you start orbiting (with a custom setOrbitPoint), because of the focalOffset, as your camera.getTarget is not where your camera actually points at.

I have no idea how to to adjust cameraControls position, target, and focalOffset in order to dollyTo3DCursor

**Comment by yomotsu** [comment]
Thanks for using camera-controls.
If you wish to change the center position, use the `moveTo()` method which does not change the camera direction.
Also, camera controls can't animate `position` and `focalOffset`at the same time, because these are dependent on each other.

For the view-offset, consider using [`PerspectiveCamera.setViewOffset()`](https://threejs.org/docs/?q=camera#api/en/cameras/PerspectiveCamera.setViewOffset) instead.

e.g.
```js
{

  const width = window.innerWidth;
  const height = window.innerHeight;
  const offsetLeft = - 150; // in pixels
  const offsetTop = - 80;

  camera.setViewOffset( width, height, 0, 0, width + offsetLeft, height + offsetTop );
  renderer.render( scene, camera );

}
```

**Comment by stam** [comment]
Hey @yomotsu ,thanks for the quick reply.

The problem I have with moveTo is that it uses some local coordinate system where it's unclear to me how these local coordinates relate to world coordinates. 

I have created a codesandbox to help me explain my usecase better, based on your setOrbitCenter example: https://codesandbox.io/s/camera-controls-worldspace-7dl2e3?file=/src/index.js:2846-2860

How would I implement `moveAlongZAxis` to move 1 unit along the world z-axis like it's currently doing, but without changing the camera direction.
I also would like this to work after I changed the orbitCenter.
